molecule/mysql.8.4/prepare.yml

```
@@ -0,0 +1,14 @@
+---
+
+- name: Prepare
+  tags: [always]
+  hosts:
+    - debian.bullseye
+    - debian.bookworm
```

```suggestion
  hosts:
    - debian
    - "!debian.buster"
```
Permet de simplifier la gestion des releases:
- plus simple pour trouver les cas spéciaux buster
- supporte de fait les futures releases

(à appliquer de partout, bien entendu :) )

---

Partout, genre sur tout les rôles ?

---

nan, juste pour les tests MySQL 8.4 :)